### PR TITLE
Backup phrase improvements

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
This PR contains improvements to use backup phrase flow.

- The ordering of generated words is changed. It now increases from left to right.
- The layout of the generated words is changed. The number label is aligned to center left and text is centered to the available space.
- At least one item from each page is selected for verification
- Text fields are made compatible with large text scales

### CODE REVIEW

File: lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart [hunk 1]
```diff
@@ -173,11 +169,31 @@ class VerifyBackupPhrasePageState extends State<VerifyBackupPhrasePage> {
   _selectIndexes() {
     List list = List.generate(23, (i) => i);
     list.shuffle();
-    for (var i = 0; i < 3; i++) {
+    for (var i = 0; _randomlySelectedIndexes.length < 3; i++) {
```
Comment (roeierez) [inline lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart]: too much code (and unefficient) to generate 3 random numbers.To generate a number from page N where M is the number of words per page you can use the following:

random.NextInt(M) + M × N

Comment (erdemyerebasmaz) [reply lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart]: [Fixed](https://github.com/breez/breezmobile/pull/180/commits/58598442b87dcde113ce27f808709656a1619364)

File: lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart [hunk 2]
```diff
@@ -171,14 +169,28 @@ class VerifyBackupPhrasePageState extends State<VerifyBackupPhrasePage> {
...
+    var n1 = Random().nextInt(12);
+    var n2 = Random().nextInt(12) + 12;
+    var n3 = Random().nextInt(22);
+    if (n3 >= n1) n3++;
+    if (n3 >= n2) n3++;
+    _randomlySelectedIndexes.addAll([n1, n2, n3]);
     _randomlySelectedIndexes.sort();
   }
 
+  bool hasRange(int start, int end, List numbers) {
```
Comment (roeierez) [inline lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart]: can remove this method now.

Comment (erdemyerebasmaz) [reply lib/routes/shared/security_pin/backup_phrase/verify_backup_phrase_page.dart]: [Done](https://github.com/breez/breezmobile/pull/180/commits/82ef333d135368f9b757c3608a3ba7cb19fa6c99)

**Pull request merged**